### PR TITLE
chore(build): add support for multiarch build

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

**What this PR does?**:

This PR adds support for multi-arch builds for cstor-base and cstor images.

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Has the change log section been updated?
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track:
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them:

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: ci
+
+on:
+  pull_request:
+    branches:
+      # on pull requests to master and release branches
+      - master
```
Comment (mynktl) [inline .github/workflows/pull_request.yml:21]: this should be develop branch.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        uses: actions/checkout@v2
+
+      - name: Shellcheck
+        uses: reviewdog/action-shellcheck@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
+          path: '.'
+          pattern: '*.sh'
+          exclude: './vendor/*'
```
Comment (mynktl) [inline .github/workflows/build.yml:32]: not required

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    needs: ['lint']
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set tag
+        run: |
+          BRANCH="${GITHUB_REF##*/}"
+          CI_TAG=${BRANCH#v}-ci
+          if [ ${BRANCH} = "master" ]; then
```
Comment (mynktl) [inline .github/workflows/build.yml:45]: develop branch

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    needs: ['lint', 'cstor-base']
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set tag
+        run: |
+          BRANCH="${GITHUB_REF##*/}"
+          CI_TAG=${BRANCH#v}-ci
+          if [ ${BRANCH} = "master" ]; then
```
Comment (mynktl) [inline .github/workflows/build.yml:81]: develop branch

**Comment by shubham14bajpai** [comment]
Moved this PR to licstor repo with ref to the latest refactoring. https://github.com/openebs/libcstor/pull/71

**Pull request closed without merging**